cmd/scw/testdata/test-all-usage-account-ssh-key-add-usage.stderr.golden

```diff
@@ -10,6 +10,7 @@ EXAMPLES:
 ARGS:
   [name]              The name of the SSH key
   public-key          SSH public key. Currently ssh-rsa, ssh-dss (DSA), ssh-ed25519 and ecdsa keys with NIST curves are supported
+  [project-id]        Project owning the resource
```

Comment:
@remyleone we could add a `(beta)` notice at the end of the doc for this. What do you think ?